docs/modules/ROOT/pages/how-tos/creating.adoc

```diff
@@ -1 +1,131 @@
-= Creating applications and components
\ No newline at end of file
+= Creating applications and components
+
+To create an application in {ProductName}, someone must enable build pipelines in your instance of {ProductName}. At the time of publication, this process includes configuring a smee channel, to listen for users' pull requests, and creating a GitHub App, so {ProductName} can access those PRs. The first procedure in this document explains how to perform these tasks. 
+
+Once build pipelines are enabled, users can create their applications in {ProductName}. The second procedure below describes how to create an application. 
+
+== Enabling build pipelines for your instance of {ProductName}:
+
+.Prerequisites:
+
+* Install link:https://github.com/konflux-ci/konflux-ci/tree/main[{ProductName}]
+* Read/write access to your {ProductName} instance’s repository
+
+.Procedure:
+
+. Start a new link:https://smee.io/[smee] channel.
+. In the repository for your instance of {ProductName}, edit the file /smee/smee-client.yaml. Replace `<smee-channel>` with the webhook proxy URL from the previous step. 
+. Create a GitHub App according to link:https://pipelinesascode.com/docs/install/github_apps/#manual-setup[these Pipelines as Code instructions]. For the *Webhook URL* field, use the webhook proxy URL for your smee channel.
+
+== Creating your applications and components with {ProductName}:
+
+At the time of publication, to create applications in {ProductName}, you need to manually maintain their configurations by editing YAML files. This might change, in favor of maintaining those configurations in the {ProductName} UI.
+
+.Prerequisites:
+
+* link:https://kubernetes.io/docs/tasks/tools/[kubectl] CLI tool
+* The following values from your {ProductName} GitHub App:
+** The private key
+** The App ID
+** The webhook secret
+* A GitHub repository with source code for your application 
+** It must have the same basic structure as our example repository
+* Read/write access to the repository for your instance of {ProductName} 
+
+.Procedure:
+
+. In the GitHub repository for your application, install your organization’s {ProductName} GitHub App.
+. In the repository for your instance of {ProductName}, prepare an `application-and-component.yaml` file for your namespace. 
+.. If it your namespace does not already have this YAML file, create it with the following path: `/test/resources/demo-users/user/ns2/application-and-component.yaml`. Use the namespace 2 file as a template.
```

Comment:
```suggestion
.. If your namespace does not already have this YAML file, create it with the following path: `/test/resources/demo-users/user/ns2/application-and-component.yaml`. Use the namespace 2 file as a template.
```

Reply:
We're mentioning to create the file in ns2 path while we're also asking to user namespace 2 as a reference.

Reply:
Ah yep my bad. I replaced "ns2" with "<namespace>" so I think it makes sense now. Is there anything else? Can we merge the doc?

Reply:
Not sure I'm the authority here, but I'm ok with merging this.

Reply:
Ok--if I was supposed to ask someone else, that'll be on me. But if the doc looks technically accurate to you, then I'm pretty confident that's all we need. Thanks!